src/main/java/com/simondata/pouroversql/clients/SftpEngine.java

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019-present, Simon Data, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.simondata.pouroversql.clients;
+
+import org.apache.commons.lang3.EnumUtils;
+
+/**
+ * <h1>SFTPEngine</h1>
+ * The type of SFTP client to connect to.
+ */
+public enum SftpEngine {
```

Comment:
Is this the same as the other enum class?

Reply:
yeah; went ahead and combined into one